package.json

```diff
@@ -56,8 +57,10 @@
         "express": "^4.18.2",
         "file-loader": "^6.2.0",
         "http-server": "^14.1.1",
+        "husky": "^7.0.0",
```

Comment:
Why did you choose v7 instead of v8?

Reply:
Bumped to latest version(8.0.3) in d2077c3c7d0158f67837eda6e8400712ad2b4c7e. Well `npx husky-init && npm install` command installs v7